Prettify/neatify README

shayanhabibi opened the pull request:
Some small grammatical corrections to improve the professional presentation of the document to english readers.

shayanhabibi committed:
Prettify/neatify README

Some small grammatical corrections to improve the professional presentation of the document.

bung87 commented on README.md:
```diff
@@ -1,25 +1,34 @@
-then use `newScorper` proc and pass `isSecurity = true`,`privateKey`, `certificate`  
+By default, exceptions will be raised and exit the program.
+
+You can disable this with the compile switch `-d:chronosStrictException`. See chronos [#Error handling](https://github.com/status-im/nim-chronos#error-handling) for more details.
```
this flag does not disable the behavior, it just do strict check and log out where will cause exception trigger the behavior.

shayanhabibi replied:
Ah I think I understand now; I hadn't actually looked at the chronos documentation for the exception handling before. Good to know now XD; I've updated the readme to just include a few snippets from their document and then moved it down the readme.

bung87 commented:
other parts good to me , Thanks !

shayanhabibi committed:
corrected -d:chronosStrictException expl

Replaced with direct snippets from chronos page followed by a link to the explanation.

Moved section further down as I believe the features/handling of scorper should be demonstrated before the features/handling of chronos
